### PR TITLE
/nordic-events/ -> /events/

### PR DESCRIPTION
**Pull request by bast**
At the same time (in my opinion) simplified the Jekyll
source layout. I think it is simpler to not use Jekyll collections
for events at this point.

### CODE REVIEW

**Comment by naoe-tatara** [comment]
Already put the link to indico registration page?

**Comment by eglerean** [comment]
I tested it and it looks good to merge to me. I am not sure about the link to indico that @naoe-tatara mentioned, if that is not an issue then this is ok to merge.

**Comment by naoe-tatara** [comment]
Sorry for an unclear expression; I meant that we could put the link to the registration already in the page instead of 

### Registration

There will be a lightweight registration form to get
connection details.

In the events/2020-online-get-together.md?

File: events/2020-online-get-together.md [hunk 1]
```diff
@@ -1,27 +1,25 @@
...
 session on Nov 30th, followed by days of talks on Dec 1-2 (mornings
 for workshops and talks, afternoons for discussions).
 
-<!--break-->
 
-#### Registration ####
+### Registration
 
 There will be a lightweight registration form to get
 connection details.
```
Comment (naoe-tatara) [inline events/2020-online-get-together.md:20]: Registration [here](https://indico.neic.no/event/146/)

Comment (bast) [reply events/2020-online-get-together.md:20]: Adding ...

Comment (bast) [reply events/2020-online-get-together.md:19]: ... added

File: events/2020-online-get-together.md [hunk 2]
```diff
@@ -51,7 +48,8 @@ Wednesday, 2 December:
 * 12:30 **Discussion sessions**
 * 15:00 Close
 
-#### Proposal submissions ####
+
+### Proposal submissions
 
 There will be a lightweight abstract
 submission system, if you would like to submit events.
```
Comment (naoe-tatara) [inline events/2020-online-get-together.md:55]: [Abstract submission form](https://indico.neic.no/event/146/overview) is open. Submission deadline: 9th November 2020, 23:59 CET.

**Pull request merged**